### DISCUSSION TITLE
docker部署1.9.1版本，初始化数据库报错1366, Incorrect string value

### DISCUSSION THREAD

**Discussion by JaySem** [hhyo/Archery#2214] [Q&A]
执行到python3 manage.py migrate  报错
MySQLdb._exceptions.OperationalError: (1366, "Incorrect string value: '\\xE6\\x85\\xA2\\xE6\\x97\\xA5...' for column 'name' at row 1")
django.db.utils.OperationalError: (1366, "Incorrect string value: '\\xE6\\x85\\xA2\\xE6\\x97\\xA5...' for column 'name' at row 1")

查看my.cnf中默认已经配置了utf8mb4,请问这类报错是什么原因？

**Comment by JuneyQiu** [comment]
hello ,have you resolved it  ?I encounter the same problem;
